products/sle/main.pm

```diff
@@ -1001,8 +1001,12 @@ sub load_online_migration_tests() {
 
 sub load_fips_tests_web() {
     loadtest "console/curl_https.pm";
+    loadtest "console/wget_https.pm";
 }
 
+sub load_fips_tests_encrypt() {
+    loadtest "console/journald_fss.pm";
+}
 sub prepare_target() {
```

Comment:
There should be a blank line separating the sub definition from the following code